src/GoTrueClient.ts

```diff
@@ -2388,11 +2385,6 @@ export default class GoTrueClient {
           return { data: null, error }
         }
 
-        // TODO: Remove once: https://github.com/supabase/auth/pull/1717 is deployed
-        if (params.factorType === 'phone') {
```

Comment:
TOTP object should now be null since v2.158.1 is deployed  (we are now on v2.160.0)

To verify this go toSettings > Infrastructure